docs/contributor/image_mode.md

```diff
@@ -0,0 +1,369 @@
+image from the `registry.redhat.io` registry
+* The build `USER_PASSWD` argument is used to set a password for the `redhat` user
+
+```
+PULL_SECRET=~/.pull-secret.json
+USER_PASSWD=<your_redhat_user_password>
+IMAGE_NAME=microshift-4.16-bootc
+
+sudo podman build --authfile "${PULL_SECRET}" -t "${IMAGE_NAME}" \
+    --build-arg USER_PASSWD="${USER_PASSWD}" \
+    -f Containerfile
+```
+
+Verify that the local MicroShift 4.16 `bootc` image was created.
+
+```
+$ sudo podman images "${IMAGE_NAME}"
+REPOSITORY                       TAG         IMAGE ID      CREATED        SIZE
+localhost/microshift-4.16-bootc  latest      193425283c00  2 minutes ago  2.31 GB
+```
+
+### Publish Image
+
+Run the following commands to log into a remote registry and publish the image.
+
+> The image from the remote registry can be used for running the container on
+> another host, or when installing a new operating system with the `bootc`
+> image layer.
+
+```
+REGISTRY_URL=quay.io
+REGISTRY_IMG=myorg/mypath/"${IMAGE_NAME}"
+
+sudo podman login "${REGISTRY_URL}"
+sudo podman push localhost/"${IMAGE_NAME}" "${REGISTRY_URL}/${REGISTRY_IMG}"
+```
+
+> Replace `myorg/mypath` with your remote registry organization name and path.
+
+## Run MicroShift Bootc Image
+
+Log into the `RHEL 9.4 host` using the user credentials that have SUDO
+permissions configured.
+
+### Configure CNI
+
+The MicroShift CNI driver (OVN) requires the Open vSwitch service to function
+properly. The service depends on the `openvswitch` kernel module to be available
+in the `bootc` image.
+
+Run the following commands on the host to check the `openvswitch` module presence
+and the version of `kernel-core` package used in the `bootc` image. Note that the
+kernel versions are different.
+
+```
+$ find /lib/modules/$(uname -r) -name "openvswitch*"
+/lib/modules/6.9.9-200.fc40.x86_64/kernel/net/openvswitch
+/lib/modules/6.9.9-200.fc40.x86_64/kernel/net/openvswitch/openvswitch.ko.xz
+
+$ IMAGE_NAME=microshift-4.16-bootc
+$ sudo podman inspect "${IMAGE_NAME}" | grep kernel-core
+        "created_by": "kernel-core-5.14.0-427.26.1.el9_4.x86_64"
+```
+
+When a `bootc` image is started as a container, it uses the host kernel, which is
+not necessarily the same one used for building the image. This means that the
+`openvswitch` module cannot be loaded in the container due to the kernel version
+mismatch with the modules present in the `/lib/modules` directory.
+
+One way to work around this problem is to pre-load the `openvswitch` module before
+starting the container as described in the [Run Container](#run-container) section.
+
+### Configure CSI
+
+If the host is already configured to have a `rhel` volume group with free space,
+this configuration is inherited by the container so that it can be used by the
+MicroShift CSI driver to allocate storage.
+
+Run the following command to determine if the volume group exists and it has the
+necessary free space.
```

Comment:
Is 2G enough free space ? Not to get into the weeds, but is there a general rules like you need 90% free space ?